index.html

```diff
@@ -862,11 +862,11 @@ <h4>Create Proof (eddsa-jcs-2022)</h4>
 
             <ol class="algorithm">
               <li>
-Set <var>options</var>.<var>@context</var> to
-<var>unsecuredDocument</var>.<var>@context</var>.
+Let |proof| be a clone of the proof options, |options|.
               </li>
               <li>
-Let |proof| be a clone of the proof options, |options|.
+Set <var>proof</var>.<var>@context</var> to
+<var>unsecuredDocument</var>.<var>@context</var>.
               </li>
               <li>
 Let |proofConfig| be the result of running the algorithm in
```

Comment:
`proof` has to be passed instead of `options` here to properly canonicalize it with the added `@context` property from the unsecured doc.

Reply:
@dlongley Missed that. Fixed. Thanks.